.gitignore

```diff
@@ -134,6 +134,7 @@ dmypy.json
 
 # Model specification files
 src/vivarium_census_prl_synth_pop/model_specifications/*.yaml
+src/vivarium_census_prl_synth_pop/model_specifications/branches/*.yaml
```

Comment:
Why are these being git-ignored?

Reply:
I assume to prevent folks from continually and accidentally adding their test yamls? I saw that the model_spec was already being ignored so followed suit w/ the scenarios file.